rs/registry/canister/src/registry.rs

```diff
@@ -502,27 +453,17 @@ mod tests {
     /// This should bring back the registry in a state indistinguishable
     /// from the one before calling this method.
     fn serialize_then_deserialize(registry: Registry) {
```

Comment:
The rest of the changes in this file are just test reworks to account for deleted/not used in production functionality.